src/raftstore/store/fsm/metrics.rs

```diff
@@ -22,6 +22,12 @@ lazy_static! {
         exponential_buckets(1.0, 2.0, 20).unwrap()
     )
     .unwrap();
+    pub static ref BATCH_FETCH_INTERVAL: Histogram = register_histogram!(
+        "tikv_raftstore_batch_fetch_interval",
```

Comment:
What's the purpose of this metrics? It looks strange as it is unpredictable. A large value is not always worse/better than a small value.

Reply:
The metrics helps to locate the problem. Although for most situations the metrics is unpredictable (`500us` is not worse/better than `5ms`), but `5ms` is very different from `50ms`. I think the metrics is still helpful for this case.
Or we can know the raftstore thread is blocked at other place or here if proposal wait time grows up?

Reply:
If it's about to recording the interval, then it should be recorded before fetching. If it's recorded before fetching, what's the difference from `PEER_RAFT_PROCESS_DURATION`?

Reply:
Em... yes they are almost same. Thank you!